### DISCUSSION TITLE
关于实验四给分

### DISCUSSION THREAD

**Discussion by LunarJune** [arcsysu/SYsU-lang#54] [Q&A] [answered]
请问实验四通过四个算例能得到多少分呢，假如在排行榜上排名倒数的话

**Comment by wu-kan** [comment, marked as answer]
update：这里的九十指的是总评九十…包括了前三个实验的完成情况。实验四自身基础得分就是你们的 performance x 100，未满 100 的分数奖励给感兴趣的同学做可选的挑战选项或是提高 performance。

---

未定，九十以上应该有吧

**Reply by LunarJune** [reply]
感谢师兄回复。
再问一下，json::parse 瓶颈问题算挑战选项是吗，解决了有加分不

**Reply by wu-kan** [reply]
> 感谢师兄回复。 再问一下，json::parse 瓶颈问题算挑战选项是吗，解决了有加分不

是，有。
